Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollNodeWidget.cpp

```diff
@@ -154,6 +157,24 @@ namespace EMotionFX
                 m_ragdollNodeCard->show();
                 m_jointLimitWidget->show();
                 m_collidersWidget->show();
+
+                RagdollManipulatorData ragdollManipulatorData;
+                auto* actorInstance = GetActorInstance();
+                auto* selectedNode = GetNode();
+                if (GetActor() && actorInstance && selectedNode)
+                {
+                    const Transform& nodeWorldTransform = actorInstance->GetTransformData()->GetCurrentPose()->GetModelSpaceTransform(selectedNode->GetNodeIndex());
+                    ragdollManipulatorData.m_nodeWorldTransform = AZ::Transform::CreateFromQuaternionAndTranslation(nodeWorldTransform.m_rotation, nodeWorldTransform.m_position);
+                    ragdollManipulatorData.m_colliderNodeConfiguration = colliderNodeConfig;
+                    ragdollManipulatorData.m_actor = GetActor();
+                    ragdollManipulatorData.m_node = GetNode();
+                    ragdollManipulatorData.m_collidersWidget = m_collidersWidget;
+                    ragdollManipulatorData.m_valid = true;
+                }
+                if (Integration::CVars::emfx_ragdollManipulatorsEnabled)
+                {
+                    m_ragdollViewportUiCluster.CreateClusterIfNoneExists(ragdollManipulatorData);
+                }
```

Comment:
All the code creating `ragdollManipulatorData` is not used unless `emfx_ragdollManipulatorsEnabled` is true at the end. I would suggest to surround all with the check that `emfx_ragdollManipulatorsEnabled` is true, that way it doesn't have to do any unnecessary operations when it's false.

Reply:
Good call, moved the check on the cvar to the beginning.